types/openpgp/index.d.ts

```diff
@@ -38,7 +38,7 @@ export interface EncryptOptions {
 
 export interface EncryptedMessage {
     data: string,
-    message: string,
+    message: message.Message,
```

Comment:
Nice catch 😄 
grr, code duplication..
Hopefully they're the same now!

Reply:
@weswigham points out that you can make a common directory that shares code between the two versions. Then you just have the hassle of separating the differences between the two versions from the common code.

Reply:
i'm not sure how you mean @sandersn , is there an example? 😇

Reply:
I don't think there is an example yet, and I *definitely* haven't done it myself, but you'd do something like

```
+ openpgp
  - index.d.ts
  - openpgp-tests.ts
  + ts3.2
  |  - index.d.ts
  |  - openpgp-tests.ts
  + common
  |  - index.d.ts
```

And then in openpgp/index.d.ts, you'd start with

```ts
export { Bunch, Of, Stuff } from './common/index'
export interface DifferentThing {
}
```

ts3.2 would be the same, except it would get its common parts from '../common/index'.